src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -106,7 +106,9 @@
 
             // tell our visualforce page we are done loading
             var event = $A.get("e.c:HH_ContainerLoadedEvent");
-            event.fire();
```

Comment:
basically i'm trying to keep it matching what would have been the correct event based design, if Lightning allowed visualforce pages to register their event handlers.  so yes, I prefer to keep it as is.